client/ayon_blender/api/ops.py

```diff
@@ -418,6 +431,11 @@ def draw(self, context):
         layout.separator()
         layout.operator(SetFrameRange.bl_idname, text="Set Frame Range")
         layout.operator(SetResolution.bl_idname, text="Set Resolution")
+        project = os.environ.get("AYON_PROJECT_NAME")
+        settings = get_project_settings(project).get("blender")
+        unit_scale_settings = settings.get("unit_scale_settings")
+        if unit_scale_settings.get("enabled"):
+            layout.operator(SetUnitScale.bl_idname, text="Set Unit Scale")
```

Comment:
This seems odd actually - shouldn't the menu entry itself ALWAYS be present - even when not enabled to "automatically set"?
```suggestion
        layout.operator(SetUnitScale.bl_idname, text="Set Unit Scale")
```